vortex-file/src/writer.rs

```diff
@@ -90,6 +91,19 @@ impl VortexWriteOptions {
 }
 
 impl VortexWriteOptions {
+    /// Prioritize small size over read-throughput and read-latency.
+    pub fn compact() -> Self {
+        Self {
+            strategy: WriteStrategyBuilder::new()
+                .with_compressor(CompactCompressor::default())
+                .build(),
+            exclude_dtype: false,
+            file_statistics: PRUNING_STATS.to_vec(),
+            max_variable_length_statistics_size: 64,
```

Comment:
Shall we make a constant for this, it seems random here

Reply:
I opted to use `Self::default().with_compressor(...)`.